packages/paste-core/components/input-box/__tests__/input-chevron-wrapper.test.tsx

```diff
@@ -2,20 +2,47 @@ import * as React from 'react';
 
 import {render, screen} from '@testing-library/react';
 import {CustomizationProvider} from '@twilio-paste/customization';
-import {InputChevronWrapper} from '../src';
+import {getInputChevronIconColor, InputChevronWrapper, InputBox} from '../src';
 
 describe('HTML attributes', () => {
   it('should set a element data attribute for InputChevronWrapper', () => {
-    render(<InputChevronWrapper element="INPUT_CHEVRON_WRAPPER">input-chevron</InputChevronWrapper>);
+    render(
+      <InputBox element="INPUT_BOX">
+        <InputChevronWrapper element="INPUT_CHEVRON_WRAPPER">input-chevron</InputChevronWrapper>
```

Comment:
Issue: For testing the default element attribute, this shouldn't have an element set on it (the one for `InputBox` isn't necessary either, but it's now what you're testing so :shrug-emoji:). Also, it looks like `InputChevronWrapper`'s default element attribute is just `CHEVRON_WRAPPER`. I personally prefer `INPUT_CHEVRON_WRAPPER` tho.

Reply:
There's a comment in the code about this, element is always required on this component:

```
  // Requiring element here instead of extending directly from BoxProps.
  // This ensures an element prop is always passed on these composite components.
```

Reply:
ah okay. I mean, in that case, this test and the following one are repeats. I would still suggest changing the wrapper's default element to `INPUT_CHEVRON_WRAPPER` though, in keeping w/ naming conventions of subcomponents.